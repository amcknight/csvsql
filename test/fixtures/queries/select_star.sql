SELECT *
FROM test/fixtures/tables/people
WHERE age > 30
