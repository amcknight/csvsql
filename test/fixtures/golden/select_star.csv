Andrew, McKnight, 35
Tom, Ash, 36
Cayley, Carlson, 31
